packages/api/src/validators/community.ts

```diff
@@ -1,7 +1,38 @@
 import { Joi, celebrate } from 'celebrate';
 
+import { ContainerTypes, ValidatedRequestSchema } from '~utils/queryValidator';
+import { config } from '@impactmarket/core';
```

Comment:
The config here is used for `defaultLimit` and `defaultOffset`. This are api-specific, so avoid using the core config.